src/screens/ChatScreen/index.js

```diff
@@ -89,6 +89,7 @@ const ChatScreen = ({ route }) => {
         renderAvatar={null}
         alignTop
         renderBubble={props => <ChatBubble {...props} />}
+        renderTime={() => {}}
```

Comment:
This is needed?

Reply:
I had to remove the default timeStamp, since I used a custom one, and this was the way I find to do it

Reply:
This might be alittle confusing. If not needed I would recommend to remove it 2 weeks from now you might not have a clue why this was done